packages/core/src/cli/commands.ts

```diff
@@ -56,6 +56,7 @@ export async function init({
     const { content: config, filePath: configFilePath } = await loadConfigV2({
       cwd: root,
       path: commonOpts.config,
+      mode: commonOpts.envMode,
```

Comment:
The mode and envMode are two different concept, so we want to distinguish between these two concepts.

- **mode**: currently uses NODE_ENV, equivalent to webpack’s mode concept, basically used to set some build configurations for a specific environment.

- **env mode**: can be set via `--env-mode <value>`, only be used to load `.env.[env-mode]` file. Can be set to any value.

Reply:
At the same time, I am worried that exposing envMode in rsbuild.config.ts is a potential burden for users who use the Rsbuild JavaScript API. 

They also need to pass in this value to `loadConfig` method to ensure that rsbuild.config.ts can get this value.

Reply:
Got it, what I want to add is `envMode`